dbus-data/org.fedoraproject.Anaconda.Addons.HelloWorld.conf

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE busconfig PUBLIC
+ "-//freedesktop//DTD D-BUS Bus Configuration 1.0//EN"
+ "http://www.freedesktop.org/standards/dbus/1.0/busconfig.dtd">
+<!--
+This file defines access policies for the D-Bus object of the Hello World addon. Since D-Bus uses
+CamelCase names, it is "HelloWorld" here. The part "org.fedoraproject.Anaconda.Addons" is a given
+for addons. See also the file constants.py.
```

Comment:
@poncovka  "org.fedoraproject.Anaconda.Addons" is the addon interface I think ?

Reply:
In this context, it is the namespace that all addons should use. Anaconda starts all services in this namespace by default.